themes/mlibrary_new/javascripts/image_viewer.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', function () {
+    var $map = document.querySelector("#image-zoomer-os");
+    var $toolbar;
+
+    var identifier = $map.dataset.identifier;
+    var image_base;
+
+    var viewer; var mode;
+    // -- info_url is the URL to the IIIF info end point for the image
+    var info_url = identifier;
+
+    viewer = OpenSeadragon({
+        id: "image-zoomer-os",
+        prefixUrl: "//openseadragon.github.io/openseadragon/images/",
+        gestureSettingsMouse: {
+          scrollToZoom: false,
+          clickToZoom: false,
+          dblClickToZoom: true,
+          flickEnabled: true,
+          pinchRotate: true
+        },
+        gestureSettingsTouch: {
+          pinchRotate: true
+        },
+        showNavigationControl: true,
+        showRotationControl: true,
+        zoomInButton: 'action-zoom-in',
+        zoomOutButton: 'action-zoom-out',
+        rotateLeftButton: 'action-rotate-left',
+        rotateRightButton: 'action-rotate-right',
+        homeButton: 'action-reset-viewer'
+    });
+
+viewer.addHandler('zoom', function(e) {
```

Comment:
What's going on with 34-36?  If we're not showing the zoom factor, can we just remove them?